catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/DurableFileComponent.java

```diff
@@ -22,9 +22,6 @@
 import org.apache.commons.lang.StringUtils;
 
 public class DurableFileComponent extends GenericFileComponent<EventfulFileWrapper> {
-  Boolean isDav;
```

Comment:
Removed and pushed down into the DurableFileEndpoint since this is a singleton from camels perspective.